client/src/utils/constants.ts

```diff
@@ -29,6 +29,256 @@ const contributorPages = [
   wizardPages.ONBOARD_5,
 ];
 
+const neighborhoods = [
```

Comment:
Can we put this in another file for itself LOL, and then we can just import it into constants and then export it directly from there so that it can still be imported from constants